flask_project/campaign_manager/templates/campaign_detail.html

```diff
@@ -110,6 +110,8 @@ <h3 class="detail-campaign-name">{{ name }}</h3>
                             data-layout="button_count">
                         </div>
                         <button class="btn btn-info btn-manage pull-right" onclick="location.href='/campaign_manager/edit/{{ uuid }}'">Manage</button>
+                        <button class="btn btn-info btn-open-app" onclick="openOtherDataCollectonForm(this)">Maps.me</button>
```

Comment:
this should have some conditional, logic around it. I.e. only when it is available in the project.